lib/services/php-file-parser.coffee

```diff
@@ -273,6 +280,8 @@ module.exports =
       element = element.replace /^\s+|\s+$/g, ""
       if element[0] == '{' or element[0] == '(' or element[0] == '['
         element = element.substring(1)
+      else if element.indexOf('return ') == 0
```

Comment:
I'm using getStackClasses on every type of function call in the function goto. This means that both static and local calls are being passed. `getStackClasses` wasn't coping with there not being a $ sign to stop the backwards search. So when you have a function call that isn't called from a $ variable (e.g. $this) then it keeps going backwards up the editor looking for the $. If it hits a `{` it stops and returns from that buffer position to the starting buffer position.

Example:
```
{

        return parent::foo();
```

I adjusted it so that it returned when the next character it's going to look at is `{`.

It then returned:
```

      return parent::foo();
```

Finally I trim the text and return that. Now when it comes to parsing the stack classes the return is still in there. So line 283 and 284 are removing that return if it is found.